hpotter/__main__.py

```diff
@@ -1,46 +1,73 @@
-import importlib
 import signal
 
 import hpotter.plugins
-from hpotter.plugins.plugin_handler import start_server, stop_server
+from hpotter.plugins.plugin_handler import start_server, stop_server, stop_all_running_containers
 from hpotter.env import logger, stop_shell, close_db
 
 plugins_dict = hpotter.plugins.__dict__
 
 def shutdown_servers(signum, frame):
-    for plugin_name in plugins_dict['__all__']:
+    '''
```

Comment:
Remove all these commented out testing methods. If you think they are useful, I don't think we need them, add a comment explaining why we still need them